public/electron.js

```diff
@@ -75,15 +75,35 @@ const edit = [
           label: 'GDLauncher',
           submenu: [
             {
-              label: 'Hide',
+              label: `About GDLauncher`,
+              role: 'about'
+            },
+            { type: 'separator' },
+            {
+              label: 'Services',
+              role: 'services',
+              submenu: []
+            },
+            { type: 'separator' },
+            {
+              label: `Hide GDLauncher`,
               accelerator: 'Command+H',
-              selector: 'hide:'
+              role: 'hide'
+            },
+            {
+              label: 'Hide Others',
+              accelerator: 'Command+Alt+H',
+              role: 'hideOthers'
+            },
+            {
+              label: 'Show All',
+              role: 'unhide'
             },
             { type: 'separator' },
             {
-              label: 'Quit',
+              label: `Quit GDLauncher`,
```

Comment:
No need for the interpolation quote here, just use a single quote

Reply:
Oh, sorry, forgot to press the alt key on the keyboard while writing the quote 👍 I’ll fix that later (if you want you should be able too)

Reply:
Actualy happend three times :/